Clean up `ModalCard::class`. (#59)
yiisoft/yii-bulma: docs/modalcard.md

        </section>
        <footer class="modal-card-foot">
            <button class="button is-success">Save changes</button>
            <button class="button is-danger is-outline">Cancel</button>
        </footer>
    </div>
</div>
```

## Setters

All setters are immutable and return a new instance of the `Yiisoft\Yii\Bulma\ModalCard` class with the specified value.

Method | Description | Default
-------|-------------|---------
`attributes(array $value)` | The HTML attributes. | `[]`
`autoIdPrefix(string $value)` | Prefix to the automatically generated widget ID. | `w`
`backgroundClass(string $value)` | The class for the modal background. | `modal-background`
`bodyAttributes(array $value)` | HTML attributes for the body tag.| []
`bodyClass(string $value)` | The class for the modal body. | `modal-card-body`
`buttonClass(string $value)` | The class for the modal button. | `button modal-button`
`cardAttributes(array $value)` | HTML attributes for the card. | []
`closeButtonAttributes(array $value)` | HTML attributes for the close button. | []
`closeButtonClass(string $value)` | The class for the close button. | `button delete`

> `closeButtonSize(string $value)` | Size close button. Options available are: (`ModalCard::SIZE_SMALL`, `ModalCard::SIZE_MEDIUM`, `ModalCard::SIZE_LARGE`). | Default setting empty normal.

Reviewer: ```suggestion
`closeButtonSize(string $value)` | Сlose button size. Options available are: (`ModalCard::SIZE_SMALL`, `ModalCard::SIZE_MEDIUM`, `ModalCard::SIZE_LARGE`). | Default setting is "normal".
```

Author: done.

`contentClass(string $value)` | The class for the modal card content. | `modal-card-content`
`footClass(string $value)` | The class for the modal card foot. | `modal-card-foot`

> `footer(string $value)` | The footer content. | `''`

Reviewer: Are foot and footer the same thing?

Author: They are not the same, one is the content, the other is the CSS class.

Reviewer: Yes, got that. The question was if that is the same element referenced using two different names "foot" and "footer" or these are two different elements?

Author: done.

`footerAttributes(array $value)` | HTML attributes for the footer. | []
`headClass(string $value)` | The class for the modal card head. | `modal-card-head`

> `headerAttributes(array $value)` | HTML attributes for the header. | []

Reviewer: Are head and header the same thing?

Author: Could merge it into one, but it's not convenient, if you just want to change the CSS class.

Reviewer: No, I don't mean merging it to the same method. I mean naming i.e.:

```php
headerClass(string $value)
headerAttributes(array $value)
```

Author: done.

`id(string $value)` | Widget ID. | `''`

> `modalCardClass(string $value)` | The class for the modal card. | `modal-card`

Reviewer: Can we remove "modalCard" prefix?

Author: It's the CSS Class of the `ModalCard`, as I call it CardCssClass ?

Reviewer: Yes. That's better name 👍

Author: done.

> `modalCardTitleClass(string $value)` | The class for the modal card title. | `modal-card-title`

Reviewer: Do we need to add "modalCard" to all these methods?

Author: There's no need.

Author: fixed.

`titleAttributes(array $value)` | HTML attributes for the title. | []
`titleClass(string $value)` | The class for the modal card title. | `modal-card-title`
`toggleButtonAttributes(array $value)` | HTML attributes for the toggle button. | []

> `toggleButtonColor(string $value)` | Color toggle button. Options available are: (`ModalCard::COLOR_PRIMARY`, `ModalCard::COLOR_INFO`, `ModalCard::COLOR_SUCCESS`, `ModalCard::COLOR_WARNING`, `ModalCard::COLOR_DANGER`, `ModalCard::COLOR_DARK`). | `''`

Reviewer: ```suggestion
`toggleButtonColor(string $value)` | Toggle button color. Options available are: (`ModalCard::COLOR_PRIMARY`, `ModalCard::COLOR_INFO`, `ModalCard::COLOR_SUCCESS`, `ModalCard::COLOR_WARNING`, `ModalCard::COLOR_DANGER`, `ModalCard::COLOR_DARK`). | `''`
```

Author: done.

`toggleButtonId(?string $value)` | Toggle button ID. | `''`
`toggleButtonLabel(string $value)` | Toggle button label. | `'Toggle button'`

> `toggleButtonSize(string $value)` | Size toggle button. Options available are: (`ModalCard::SIZE_SMALL`, `ModalCard::SIZE_MEDIUM`, `ModalCard::SIZE_LARGE`). | Default setting empty normal.

Reviewer: ```suggestion
`toggleButtonSize(string $value)` | Toggle button size. Options available are: (`ModalCard::SIZE_SMALL`, `ModalCard::SIZE_MEDIUM`, `ModalCard::SIZE_LARGE`). | Default setting is "normal".
```

Author: done.

> `withoutCloseButton(bool $value)` | Whether the close button is disable. | `false`

Reviewer: ```suggestion
`withoutCloseButton(bool $value)` | Whether the close button is disabled. | `false`
```

Author: done.

> `withoutToggleButton(bool $value)` | Whether the toggle button is disable. | `false`

Reviewer: ```suggestion
`withoutToggleButton(bool $value)` | Whether the toggle button is disabled. | `false`
```

Author: done.
